chain/chain/src/chain.rs

```diff
@@ -1557,12 +1525,12 @@ impl Chain {
         sync_hash: CryptoHash,
         part_id: u64,
         num_parts: u64,
-        data: &Vec<u8>,
+        data: Vec<u8>,
```

Comment:
what is this change for?

Reply:
To know it's not borrowing. `set_state_part` indicates it wants to consume the data.

Reply:
But inside the function `data` is not consumed. Why do you need to pass the value?

Reply:
Yep. I guess it should but our storage seems to work different way.
I'm okay with either borrowing or not.

Reply:
Ah, I see. That's my fault. I'll revert it.